WORKSPACE

```diff
@@ -321,6 +321,10 @@ maven_install(
         "https://maven.google.com",
     ],
     jetify = True,
+    jetify_include_patterns = [
+        "startswith(com.google:)",
+        "startswith(com.android.support:)",
```

Comment:
While this is a clever trick, I think a Bazel-idiomatic approach would be listing labels explicitly:
```starlark
jetify_include = [
    "@maven//:junit_junit",
]
```

Reply:
That's a bit too much when you need to jetify entire groupId for example, but I don't disagree really

The reason why I went with pattern list is because I found it very nice when we were using OkBuck https://github.com/uber/okbuck/blob/a0eb9e9daf27a17715d238fd2ed9fd24c5ad74f9/buildSrc/src/main/java/com/uber/okbuck/extension/JetifierExtension.java#L43